### PR TITLE
Assign calls to fromComponent with colors

### PR DESCRIPTION
**Pull request by Ichbinjoe**
When the default call to fromComponent is called, sometimes the default
color is not the correct color for the context in which it is called.
Change invocations of fromComponent where this is the case to the proper
context specific default color.

Originally reported in https://github.com/drtshock/PlayerVaults/issues/398

### CODE REVIEW

**Comment by Spottedleaf** [comment]
cb/spigot issue for sure

File: Spigot-Server-Patches/0419-Assign-calls-to-fromComponent-with-colors.patch [hunk 1]
```diff
@@ -0,0 +1,285 @@
+From a16088fbd41772d9dc2ac024443088e7a821728a Mon Sep 17 00:00:00 2001
```
Comment (Spottedleaf) [inline Spigot-Server-Patches/0419-Assign-calls-to-fromComponent-with-colors.patch:1]: we require paper comments for diffs made in src

**Comment by BillyGalbreath** [comment]
This has been brought up before, and the consensus has been `wont fix` due to diverting from upstream. This is one of those things that must be fixed in CB or not at all, otherwise plugins behave differently on different implementations which is not desirable.

**Comment by Ichbinjoe** [comment]
No doubt - I've just had bad experiences with md__5 in the past. I'll close this out and keep it as a reference for the original issue but I don't really plan on porting this upstream further.

**Pull request closed without merging**